metricsbp/config.go

```diff
@@ -0,0 +1,50 @@
+package metricsbp
+
+import (
+	"context"
+	"io"
+
+	"github.com/reddit/baseplate.go/log"
+	"github.com/reddit/baseplate.go/tracing"
+)
+
+// Config is the confuration struct for the metricsbp package.
+//
+// Can be deserialized from YAML.
+type Config struct {
+	// Namespace is the standard prefix applied to all of your metrics, it should
+	// include the name of your service.
+	Namespace string
+
+	// Endpoint is the endpoint you send metrics to.
+	Endpoint string
+
+	// CounterSampleRate is the % of counters that you want to send to your metrics
```

Comment:
nit: say "%" in the comment will mislead user from putting `50` when they want to mean 50% sample rate, but that will give them 100% sampling (because 50>1), and they actually need to use `0.5` for actually 50% sample rate.